implementation.md

```diff
@@ -0,0 +1,22 @@
+# Implementation Recommendations
+
+This document describes implementation recommendations for a STAC API.
+
+## CORS
+
+It is recommended that public APIs advertise a permissive CORS configuration so UIs running on a different domain
+may more easily access them.
+
+APIs should acknowledge pre-flight request headers. In general, these header values should be set on responses:
+
+```
+access-control-allow-origin: *
+access-control-allow-methods: OPTIONS, POST, GET
+access-control-allow-headers: Content-Type
+access-control-allow-credentials: false
```

Comment:
I think this (credentials) can be removed. It doesn't need to be set to false because that's the default and we should only set what is required by STAC API itself.

Reply:
modified. I didn't realize that actually "The only valid value for this header is true (case-sensitive). If you don't need credentials, omit this header entirely (rather than setting its value to false)."